core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -9,57 +9,70 @@ import "../../oracle/interfaces/StoreInterface.sol";
 import "../../oracle/interfaces/FinderInterface.sol";
 
 
+/**
+ * @title FeePayer contract.
+ * @notice Provides fee payment functionality for all expiring multi-party contracts.
+ * contract is abstract as each derived contract that implements `FeePayer` must implement some
+ * logic around the `pfc` value.
+ */
+
 abstract contract FeePayer is Testable {
     using SafeMath for uint;
     using FixedPoint for FixedPoint.Unsigned;
     using SafeERC20 for IERC20;
 
-    /**
-     * The collateral currency used to back the positions in this contract.
-     */
+    /****************************************
+     *      FEE PAYER DATA STRUCTURES       *
+     ****************************************/
 
+    // The collateral currency used to back the positions in this contract.
     IERC20 public collateralCurrency;
 
-    /**
-     * Finder contract used to look up addresses for UMA system contracts.
-     */
+    //  Finder contract used to look up addresses for UMA system contracts.
     FinderInterface public finder;
 
-    /**
-     * Tracks the last block time when the fees were paid.
-     */
+    // Tracks the last block time when the fees were paid.
     uint public lastPaymentTime;
 
-    /**
-     * Tracks the cumulative fees that have been paid by the contract for use by derived contracts.
-     * The multiplier starts at 1, and is updated by computing cumulativeFeeMultiplier * (1 - effectiveFee).
-     * Put another way, the cumulativeFeeMultiplier is (1 - effectiveFee1) * (1 - effectiveFee2) ...
-     *
-     * For example:
-     *
-     * The cumulativeFeeMultiplier should start at 1.
-     * If a 1% fee is charged, the multiplier should update to .99.
-     * If another 1% fee is charged, the multiplier should be 0.99^2 (0.9801).
-     */
+    // Tracks the cumulative fees that have been paid by the contract for use by derived contracts.
+    // The multiplier starts at 1, and is updated by computing cumulativeFeeMultiplier * (1 - effectiveFee).
+    // Put another way, the cumulativeFeeMultiplier is (1 - effectiveFee1) * (1 - effectiveFee2) ...
+    // For example:
+    // The cumulativeFeeMultiplier should start at 1.
+    // If a 1% fee is charged, the multiplier should update to .99.
+    // If another 1% fee is charged, the multiplier should be 0.99^2 (0.9801).
     FixedPoint.Unsigned public cumulativeFeeMultiplier;
 
-    /**
-     * @notice modifier that calls payFees().
-     */
+    /****************************************
+     *              MODIFIERS               *
+     ****************************************/
+
+    // modifier that calls payFees().
```

Comment:
nit: I think modifiers should have natspec comments since they appear like a function in the docs.

Reply:
we have not done natspec on any modifiers before. not in the DVM or at any point. I can add comments to it if you want but then we should apply this to other contracts as well.